openai/azure-openai/runtime/src/main/java/io/quarkiverse/langchain4j/azure/openai/runtime/config/LangChain4jAzureOpenAiConfig.java

```diff
@@ -11,10 +11,7 @@
 import io.quarkus.runtime.annotations.ConfigDocSection;
 import io.quarkus.runtime.annotations.ConfigGroup;
 import io.quarkus.runtime.annotations.ConfigRoot;
-import io.smallrye.config.ConfigMapping;
-import io.smallrye.config.WithDefault;
-import io.smallrye.config.WithDefaults;
-import io.smallrye.config.WithParentName;
+import io.smallrye.config.*;
```

Comment:
Please revert this change - I know you did not do this yourself but it was done by the IDE, but we generally try to stay away from star imports

Reply:
Done